unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,90 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test: device D400*
+
+import time
+import pyrealsense2 as rs
+from rspy import test
+
+# This test checks Timestamp domain toggle option for color and depth sensor
+
+
+def frame_callback(frame):
+    global expected_time_stamp_domain
+    test.check_equal(int(frame.get_frame_timestamp_domain()), expected_time_stamp_domain)
+
+
+################################################################################################
+test.start("Start test")
```

Comment:
Tests need a meaningful name.
At the log you will see each test starts by it's name and if 1 fail we want to understand from the test name what functionality was broken